### PR TITLE
HERA-446 [LMS] adds mascots set up from admin panel

### PR DESCRIPTION
**Pull request by Golub-Sergey**
**Description:** adds mascots set up from admin panel

**Youtrack:** [ticket](https://youtrack.raccoongang.com/issue/HERA-446)

### CODE REVIEW

File: lms/djangoapps/hera/models.py [hunk 1]
```diff
@@ -244,3 +244,30 @@ def reset_active_course(sender, instance, *args, **kwargs):
...
+    def _instance(cls):
+        return cls.objects.first()
+
+    @classmethod
+    def onboarding_img_url(cls):
+        if getattr(cls._instance, 'onboarding', None):
+            return cls._instance.onboarding.url
+
+    @classmethod
+    def user_dashboard_img_urls(cls):
```
Comment (sendr) [inline lms/djangoapps/hera/models.py]:
```suggestion
    def user_dashboard_img_urls(cls):
        instance = cls._instance()
        if instance:
            user_dashboard = instance.user_dashboard.url if instance.user_dashboard else None
            user_dashboard_modal =  instance.user_dashboard_modal.url if instance.user_dashboard_modal else None
            return {
                'user_dashboard': user_dashboard,
                'user_dashboard_modal': user_dashboard_modal
            }
        return {}
```

File: lms/djangoapps/hera/models.py [hunk 2]
```diff
@@ -244,3 +244,30 @@ def reset_active_course(sender, instance, *args, **kwargs):
...
+
+    _user_dashboard_images = {}
+
+    @classmethod
+    def _instance(cls):
+        return cls.objects.first()
+
+    @classmethod
+    def onboarding_img_url(cls):
+        if getattr(cls._instance, 'onboarding', None):
```
Comment (sendr) [inline lms/djangoapps/hera/models.py]:
```suggestion
        instance = cls._instance()
        if instance:
            return instance.onboarding.url if instance.onboarding else None
```

File: lms/djangoapps/hera/models.py (hunk 1 above)
Comment (Golub-Sergey) [reply lms/djangoapps/hera/models.py]: Done

File: lms/djangoapps/hera/models.py (hunk 2 above)
Comment (Golub-Sergey) [reply lms/djangoapps/hera/models.py]: Done

**Pull request merged**